spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/rdd/partitioner/ServerSideTokenRangeSplitter.scala

```diff
@@ -35,24 +39,27 @@ class ServerSideTokenRangeSplitter[V, T <: Token[V]](
         client.set_keyspace(keyspaceName)
         client
           .describe_splits_ex(tableName, startToken, endToken, splitSize.toInt)
-          .map(unthriftify(_, range.endpoints))
```

Comment:
Because this is going to be addressed by another ticket. And it requires upgrading to a new Cassandra.